open-metadata-implementation/admin-services/admin-services-api/src/main/java/org/odpi/openmetadata/adminservices/configuration/properties/SecurityOfficerConfig.java

```diff
@@ -23,21 +23,18 @@
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class SecurityOfficerConfig extends AdminServicesConfigHeader {
 
-    private int        securityOfficerServerId = 0;
-    private String     securityOfficerServerName;
-    private String     securityOfficerServerDescription;
-    private String     securityOfficerServerWiki;
+    private int         securityOfficerServiceCode = 0;
```

Comment:
These 4 values should come from the enum GovernanceServersDescription rather than be fields in the config

Reply:
I used the GovernanceServersDescription to populate these fields, but I kept those as attributes for SecurityOfficer Config class

Reply:
Hi, Mandy! Is there something specific that I need to change for this config? Where should I move these fields?

Reply:
Just remove these 4 fields from the config doc - not needed.

These fields are in the  access service config to allow access services that are not in the Egeria build to be incorporated in the open metadata repository server, managed by the admin services and serviced by the repository services.

Reply:
I removed the fields.